TROUBLESHOOTING.md

```diff
@@ -126,3 +127,45 @@ const mail = Mail.create(data);
 const body = mail.toJSON();
 console.log(body);
 ```
+
+<a name="wrapping-text"></a>
+## Wrapping Text
+
+You can write blog posts using e-mail with the help of SENDGRID API, like so:
```

Comment:
Please use `SendGrid` rather than all caps.

Reply:
@reedsa Made the changes, thank you for reviewing this one and helping me with good code practices!